schemas/README.md

```diff
@@ -0,0 +1,51 @@
+# Maintenance
+
+## Introduction
+At some point in time there will be an increase an topics or even a change to the existing topic structure. If at any point one of these happen, this will require a change to the Avro schemas in further requiring a change in the structs that represent the Avro schemas. These structs are used by the kafka-management-tool in order to map the unmarshalled JSON message from with the specified topic.
+
+## Walkthrough
+If a change is made to an existing topic structure/schema or if a new topic and Avro schema has been added this will need to be reflected in the kafka-management-tool schema struct for the corresponding Avro schema.
+
+### Addition of new topic and Avro schema
+If a new topic is to be added, there will need to be a new Avro schema file that represents the message structure. Additionally, the services that produce and consume messages from the new topic will need to create the mapping object/struct that represents the topic schema. In `Java` services this is a `Pojo` and in `Go`, a `struct`. For information on how the struct will can be created for the kafka-management-tool look below.
+Additionally, once the new struct has been created, another condition will need to be added to the `schemas.go` file to make sure that the correct struct is used when the new topic is specified in the tool.
```

Comment:
I would find this easier to read if the example was with the explanation.

Reply:
Done